src/Commands/stubs/event.stub

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace $NAMESPACE$;
+
+use $DUMMYNAMESPACE$\Event;
+use Illuminate\Queue\SerializesModels;
+use Illuminate\Contracts\Broadcasting\ShouldBroadcast;
```

Comment:
The `SerializeModels` class is included in the generated class while the `ShouldBroadCast` is left for the user to implement. Do you want that taken off ?